app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
Would be good to perform split and save result into variable. After check that lenght of this is exactly 3. Then you can do unpacking